.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "youki"]
```

Comment:
It is for the efficiency of development and debugging and will eventually be deleted.